atd-vze/src/views/Locations/Location.js

```diff
@@ -71,6 +116,25 @@ function Location(props) {
     ],
   };
 
+  const horizontalBar = {
+    labels: data.atd_txdot_locations[0].crashes_by_manner_collision
+      .map((a, index) => `${index + 1}. ${a.collsn_desc}`)
+      .map(a => formatLabel(a)),
+    datasets: [
+      {
+        label: "Number of Collisions",
+        backgroundColor: "rgba(255,99,132,0.2)",
```

Comment:
Imported the colors.js file and used the color associated with the "total crashes" widget so that they match. I made the hover color a darker green, which I also added to colors.js.